src/macros.js

```diff
@@ -233,6 +229,31 @@ defineMacro("\\thickspace", "\\;");   //   \let\thickspace\;
 //////////////////////////////////////////////////////////////////////
 // LaTeX source2e
 
+// \def\TeX{T\kern-.1667em\lower.5ex\hbox{E}\kern-.125emX\@}
+// TODO: Doesn't normally work in math mode because \@ fails.  KaTeX doesn't
+// support \@ yet, so that's omitted, and we add \text so that the result
+// doesn't look funny in math mode.
+defineMacro("\\TeX", "\\text{T\\kern-.1667em\\raisebox{-.5ex}{E}\\kern-.125emX}");
+
+// \DeclareRobustCommand{\LaTeX}{L\kern-.36em%
+//         {\sbox\z@ T%
+//          \vbox to\ht\z@{\hbox{\check@mathfonts
+//                               \fontsize\sf@size\z@
+//                               \math@fontsfalse\selectfont
+//                               A}%
+//                         \vss}%
+//         }%
+//         \kern-.15em%
+//         \TeX}
+// This code roughly aligns the top of the A with the T.  Here we approximate
+// this vertical alignment with a \raisebox{.45ex}, which seems close.
+defineMacro("\\LaTeX", "\\text{L\\kern-.36em\\raisebox{.45ex}{\\scriptsize A}" +
+                       "\\kern-.15em\\TeX}");
+
+// New KaTeX logo based on tweaking LaTeX logo
+defineMacro("\\katex", "\\text{K\\kern-.25em\\raisebox{.45ex}{\\scriptsize A}" +
+                       "\\kern-.15em\\TeX}");
```

Comment:
Does `K\\kern-.36em` even out the spaces around the `A`?

Reply:
Nice objective to even out those top spaces.  By manual search, I found `K\\kern-.16em` to balance it out pretty evenly:

![katex](https://user-images.githubusercontent.com/2218736/33233070-8d4cfc30-d1de-11e7-886c-25decf42e76c.png)
http://localhost:7936/?text=%5Cbegin%7Barray%7D%7Br%7D%5Ckatex%5C%5C%5CLaTeX%5C%5C%5CKaTeX%5Cend%7Barray%7D

I feel like the this puts the bottom of the A a little far from the K, though.  I tried tweaking the A slightly more to the left, with `K\\kern-1.17em`:

![katex](https://user-images.githubusercontent.com/2218736/33233115-0d5591f8-d1df-11e7-9643-632d31f35327.png)

Preference?  This may be nit-picky, but if we're redesigning the logo, we might as well try to get it right.  I do find these far more readable than the old logo, which is nice (but also more readable than the LaTeX logo).

Reply:
I think the top one looks the best.  In the bottom one the `A` is too cramped.

Reply:
Just to be clear, we're talking about the top logo in the top image vs. the top logo in the bottom image, right?  (Bottom logo is the same in both images, the old KaTeX logo.)

Reply:
I can't tell the difference between the top logos in the two images... whichever you think is better.